data/ecosystems/f/fenix.toml

```diff
@@ -0,0 +1,22 @@
+# Ecosystem Level Information
+title = "Fenix Protocol"
```

Comment:
hey @joeblau, thanks for the contribution. I see the Fenix is built on Ethereum. If so, please add "Fenix Protocol" to the list of sub_ecosystems on file `ethereum.toml` under `data/ecosystems/e`